Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2
+
+RUN apt-get update; \
+    apt-get install -y jmeter
+
+ADD Gemfile* /eq-performance-test/
```

Comment:
the benefit of doing this is that if none of the dependencies change then it doesn't need to do bundle install, making the build time seconds instead of minutes